### ISSUE TITLE
Larger Batch Size opinions about performance

### ISSUE THREAD

**Issue by Yashay** [open-mmlab/mmdetection#5762]
Hi,

Hope you are all well.

Thanks for such an awesome framework.

In your experience, when training with a larger batch size like 32 and adjusting the learning rate according to the Linear Scaling Rule.

Do you think performance will be negatively or positively impacted? (or remain the same)

Kind Regards

**Comment by noobying** [comment]
Hi, @Yashay , this is a question to be verified. Larger batch size is not easy to fall into suboptimal point. However, smaller batch size has more iterations to update the paramaters. This two setting has its own advantages. So it's difficult to tell which one is better. You can train your model with both setting to get a better one. Hope to help you :)

**Comment by hhaAndroid** [comment]
> Hi, @Yashay , this is a question to be verified. Larger batch size is not easy to fall into suboptimal point. However, smaller batch size has more iterations to update the paramaters. This two setting has its own advantages. So it's difficult to tell which one is better. You can train your model with both setting to get a better one. Hope to help you :)

well
